### PR TITLE
Added Ceph project

### PR DESCRIPTION
**Pull request by alastairdewhurst**
Pull Request for the STFC Ceph project.

### CODE REVIEW

File: _gsocorgs/2020/ral.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+title: "STFC Rutherford Appleton Laboratory"
+author: "Alastair Dewhurst"
+layout: default
+organization: RAL
+logo: logo_RAL.jpg
```
Comment (agheata) [inline _gsocorgs/2020/ral.md]: Please use RAL-logo.png, it exists already (naming convention)

File: _gsocprojects/2020/project_Ceph.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+---
+project: Ceph
+layout: default
+logo: ceph-logo.png
+description: |
+  [Ceph](https://ceph.io/) is a storage platform.  It implements object storage on a single distributed computer cluster, and provides interfaces for object-, block- and file-level storage. It can be used to build cloud infrastructure and web-scale object storage. Ceph aims primarily for completely distributed operation without a single point of failure, that can be run on commodity hardware and is scalable to the exabyte level.
+summary: |
```
Comment (agheata) [inline _gsocprojects/2020/project_Ceph.md]: summary: is not required (description: is used by default) , but recommended. It only makes sense when (summary < description). Either reduce summary or remove.

File: _gsocproposals/2020/proposal_LibradosStriper.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+project: Ceph
+year: 2020
+organization:
+  - RAL
+  - uglasgow
+---
+
+## Description
+
+Ceph is a scalable, self-healing, open source object storage solution used in many commercial organisations, and in academia and research. Examples of the latter include CERN, NASA, and the MeerKAT radio telescope, as well as the UK’s Rutherford Appleton Laboratory (RAL). RAL stores 25 petabytes of data originating from the CERN Large Hadron Collider in a Ceph cluster called Echo.  Data in Echo is being read at a rate of 20 - 50GB/s by analysis jobs at RAL and around the world.
```
Comment (agheata) [inline _gsocproposals/2020/proposal_LibradosStriper.md]: Direct links on keywords (CERN, NASA, MeerKAT, ..) are highly recommended.

File: _gsocorgs/2020/ral.md (hunk 1 above)
Comment (aflp91) [inline _gsocorgs/2020/ral.md]: Please, use the existing `RAL-logo.png` file

File: _gsocproposals/2020/proposal_LibradosStriper.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+The software running on analysis clusters only needs to read object data from Echo. There are often many tens of read clients accessing a data object at the same time; to enable more efficient use of Echo in this manner, RAL are proposing a way to optimise read-only data access in Ceph. This would also increase the productivity of analysis workflows accessing this experimental data,
+
+The underlying access layer used for ECHO is called libradosstriper. This C++ library allows efficient striping of objects over disks in a Ceph cluster to improve data distribution and throughput. Libradosstriper read operations currently require a write lock to be placed on the first stripe of an object (which contains metadata) to prevent modifications by other clients during the read. This is unnecessary for read-only access, and for ‘hot’ objects (those in use by many clients), this can become a bottleneck. A non-locking read would drastically reduce the number of metadata operations needed for striper reads, reduce the latency of these operations, and enable a higher throughput for reading data.
```
Comment (agheata) [inline _gsocproposals/2020/proposal_LibradosStriper.md]: emphasize "libradosstriper"

File: _gsocprojects/2020/project_Ceph.md [hunk 5]
```diff
@@ -0,0 +1,12 @@
+---
+project: Ceph
+layout: default
+logo: ceph-logo.png
```
Comment (aflp91) [inline _gsocprojects/2020/project_Ceph.md]: You have to rename this file with our new conventions: `CEPH-logo.png`

File: _gsocproposals/2020/proposal_LibradosStriper.md [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+* Test new version of libradosstriper with the multiple-read microbenchmark to show changes in performance, and show no negative effects have been introduced
+
+## Expected results
+A revised version of the libradosstriper API which optimises concurrent read-only accesses to an object.  A microbenchmark which can record statistics relating to read-only access to data objects.
+
+## Requirements
+* Essential: C++, Distributed system or client-server programming
+* Useful: Object storage orientation
+
+## Mentors
```
Comment (agheata) [inline _gsocproposals/2020/proposal_LibradosStriper.md]: Please add the mentors sorted by last name in gsoc/2020/mentors.md file

File: _gsocproposals/2020/proposal_LibradosStriper.md (hunk 4 above)
Comment (aflp91) [inline _gsocproposals/2020/proposal_LibradosStriper.md]: "ECHO" => "Echo" as previous uses?

Comment (aflp91) [reply _gsocproposals/2020/proposal_LibradosStriper.md]: And adding a link to `libradosstriper` code?

**Pull request merged**